Model/Sync/Reset/Main.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Yotpo\Core\Model\Sync\Reset;
+
+use Magento\Framework\App\Cache\TypeListInterface;
+use Magento\Framework\App\ResourceConnection;
+use Yotpo\Core\Model\Config;
+
+class Main
+{
+    const DELETE_LIMIT = 10000;
+    const UPDATE_LIMIT = 2000;
+
+    /**
+     * @var ResourceConnection
+     */
+    protected $resourceConnection;
+
+    /**
+     * @var Config
+     */
+    protected $config;
+
+    /**
+     * @var TypeListInterface
+     */
+    protected $cacheTypeList;
+
+    /**
+     * @param ResourceConnection $resourceConnection
+     */
+    public function __construct(
+        ResourceConnection $resourceConnection,
+        Config $config,
+        TypeListInterface $cacheTypeList
+    ) {
+        $this->resourceConnection = $resourceConnection;
+        $this->config = $config;
+        $this->cacheTypeList = $cacheTypeList;
+    }
+
+    /**
+     * @param int $storeId
+     * @param boolean $skipSyncTables
+     * @return void
+     * @throws \Zend_Db_Statement_Exception
+     */
+    public function resetSync($storeId, $skipSyncTables = false)
```

Comment:
`skipSyncTables` is a false-oriented naming, why not use a positive-related naming?

Reply:
Understood. We will correct this and update the PR